components/steps/style/index.less

```diff
@@ -2,7 +2,7 @@
 @import "../../style/mixins/index";
 
 @steps-prefix-cls: ~"@{ant-prefix}-steps";
-@process-icon-color: @primary-color;
+@process-icon-color: @processing-color;
```

Comment:
In 3.0 can we renaming `process` -> `processing` (or vice versa) for consistency?